src/System.Runtime.Extensions/src/System/Net/WebUtility.cs

```diff
@@ -50,6 +50,12 @@ public static string HtmlEncode(string value)
             return StringBuilderCache.GetStringAndRelease(sb);
         }
 
+        public static void HtmlEncode(string value, TextWriter output)
+        {
+            // Since we already have a private method that decodes the value using a StringBuilder we don't want to have duplicate code in order to have a better mantainance.
```

Comment:
Nit, I wouldn't bother with the comments, since they only make sense if you know there was previously an explicit implementation in terms of TextWriter. Up to you.